torch/_inductor/memory.py

```diff
@@ -0,0 +1,525 @@
+from __future__ import annotations
+
+import dataclasses
+import heapq
+import logging
+from typing import Dict, List, Set, Tuple, TYPE_CHECKING, Union
+
+from torch.utils._ordered_set import OrderedSet
+
+
+if TYPE_CHECKING:
+    from .dependencies import Dep
+
+from .ir import MultiOutputLayout
+from .scheduler import (
+    BaseSchedulerNode,
+    MemoryPlanningInfoForBuffer,
+    MemoryPlanningInfoForNode,
+    NodeUser,
+    OutputNode,
+    SchedulerBuffer,
+)
+from .utils import get_dtype_size
+from .virtualized import V
+
+
+torch_log = logging.getLogger("torch")
```

Comment:
`log = logging.getLogger(__name__)` might be better here and also more similar to how scheduler.py uses logging

Reply:
will fix.